.github/workflows/main.yaml

```diff
@@ -22,6 +22,7 @@ jobs:
         run: |
           if [ -n "$(git status --porcelain)" ]; then
             echo 'To fix this check, run "go mod tidy"'
+            git diff
```

Comment:
Please remove unused line for debug.

Reply:
I don't think it's unused. It's useful to see the diff.

Reply:
> To fix this check, run "go mod tidy"

The tip shows how to fix this issue.

Reply:
> The tip shows how to fix this issue.

In my case, the CI and local environments are different. It takes a lot of my time to find out it.

Anyway, it's very clear to print the diff. We don't need to run that command again.